### PR TITLE
save night mode in option cookie #676

### PR DESCRIPTION
**Pull request by naveed-ahmad**

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32786

File: src/components/NightModeToggle/index.js [hunk 1]
```diff
@@ -1,21 +1,34 @@
...
+    }
 
-    this.setState({ isNightMode: !this.state.isNightMode });
+    onToggle({ isNightMode: !isNightMode });
   }
 
   render() {
+    const { isNightMode } = this.props;
+
+    if (__CLIENT__ && isNightMode) {
```
Comment (mmahalwy) [inline src/components/NightModeToggle/index.js]: Probably want to just add `document.body.classList.add('night-mode');` inside of `componentDidMount(){}` if `isNightMode`. Putting it in the render will cause `document.body.classList.add('night-mode');` to continue to be called on every render

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32787

File: src/components/NightModeToggle/index.js (hunk 1 above)
Comment (naveed-ahmad) [inline src/components/NightModeToggle/index.js]: Done

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32788

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32789

**Pull request merged**